filter:
  z_min : 0.0
  z_max : 2.5
  axis : "z"
  downsample_leaf_size : 0.01

plane_removal:
  max_iterations : 200
  distance_threshold : 0.01
  table_height_min : 0.65 # 0.65
  table_height_max : 0.83 # 0.9 
  prism_z_min : 0.012
  prism_z_max : 0.15

euclidean_cluster:
  cluster_tolerance : 0.04
  min_size : 20
  max_size : 2000

color_extract:
  red_min : 330
  red_max : 10
  green_min : 75
  green_max : 160
  yelw_min : 40
  yelw_max : 70
  blue_min : 190
  blue_max : 270
  s_threshold : 0.2
  v_threshold : 0.0
